docs/ldml/tr35.md

```diff
@@ -550,17 +550,18 @@ The private use subtags listed as **excluded** in _Section 3.5.3 [Private Use Co
 
 Subtags in the region.xml file (see _Section 3.11 [Validity Data](#Validity_Data)_). These are based on [[BCP47](#BCP47)] subtag values marked as **Type: region**
 
-Unicode identifiers give specific semantics to the following subtags:
-
-|      | Name                         | Comment | ISO 3166-1 status |
-| ---- | ---------------------------- | ------- | ----------------- |
-| `QO` | Outlying Oceania             | countries in Oceania [009] that do not have a [subcontinent](https://unicode-org.github.io/cldr-staging/charts/38/supplemental/territory_containment_un_m_49.html). | private use |
-| `QU` | European Union               | **deprecated**: the _canonicalized_ form is EU | private use |
-| `UK` | United Kingdom               | **deprecated**: the _canonicalized_ form is GB | exceptionally reserved |
-| `XA` | Pseudo-Accents               | special code indicating derived testing locale with English + added accents and lengthened | private use |
-| `XB` | Pseudo-Bidi                  | special code indicating derived testing locale with forced RTL English | private use |
-| `XK` | Kosovo                       | industry practice | private use |
-| `ZZ` | Unknown or Invalid Territory | used in APIs or as replacement for invalid code | private use |
+Unicode identifiers give specific semantics to the following subtags.
+(The alpha2 codes are used as Unicode region subtags. The alpha3 and numeric codes are derived according to _Section 3.5.2 [Numeric Codes](#Numeric_Codes)_ and listed here for additional documentation.)
+
+| alpha2 | alpha3 | num | Name                         | Comment | ISO 3166-1 status |
+| ------ | ------ | --- | ---------------------------- | ------- | ----------------- |
+| `QO`   | `QOO`  | 961 | Outlying Oceania             | countries in Oceania [009] that do not have a [subcontinent](https://unicode-org.github.io/cldr-staging/charts/38/supplemental/territory_containment_un_m_49.html). | private use |
+| `QU`   | `QUU`  | 967 | European Union               | **deprecated**: the _canonicalized_ form is EU | private use |
+| `UK`   | -      | -   | United Kingdom               | **deprecated**: the _canonicalized_ form is GB | exceptionally reserved |
+| `XA`   | `XAA`  | 973 | Pseudo-Accents               | special code indicating derived testing locale with English + added accents and lengthened | private use |
+| `XB`   | `XBB`  | 974 | Pseudo-Bidi                  | special code indicating derived testing locale with forced RTL English | private use |
+| `XK`   | `XKK`  | 983 | Kosovo                       | industry practice | private use |
```

Comment:
Note that ICU has been using XXK for Kosovo (per different industry practice), as I mentioned in the ticket: https://unicode-org.atlassian.net/browse/ICU-21341?focusedCommentId=160540

Reply:
> Note that ICU has been using XXK for Kosovo (per different industry practice)

Well, I don't know what practice that is. [CLDR specifies](https://www.unicode.org/reports/tr35/#Numeric_Codes): “Unicode identifiers supply a standard mapping to these: (...) for the 3-letter codes it doubles the final letter.”

And https://github.com/unicode-org/cldr/blob/main/common%2Fsupplemental%2FsupplementalMetadata.xml#L859 has `<territoryAlias type="XKK" replacement="XK" reason="overlong"/> <!-- Kosovo -->` but no mapping for XXK.

In your comments on ICU-21341 you referred to CLDR for the Kosovo codes. Therefore, as CLDR specifies XKK, I have to assume that XXK was a typo that we should fix in ICU. --> new bug ICU-21942

Clearer documentation here in the LDML spec might have helped...

Reply:
OK, thanks, guess I will file a ticket to update ICU

Reply:
> OK, thanks, guess I will file a ticket to update ICU

I did: new bug ICU-21942

Reply:
I took that ticket and have a PR: https://github.com/unicode-org/icu/pull/2034